db/migrate/20230525081252_update_created_manually_flag_on_customers.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+class UpdateCreatedManuallyFlagOnCustomers < ActiveRecord::Migration[7.0]
+  class Customer < ApplicationRecord
+    has_many :orders, class_name: "Spree::Order"
+    acts_as_taggable
+  end
+
+  module Spree
+    class Order < ApplicationRecord
+      belongs_to :customer
+      self.table_name = 'spree_orders'
+    end
+  end
+
+  def change
+    # We want to set the created_manually flag to true for all customers that
+    #  1. have at least one tag
+    #  2. or don't have any orders
```

Comment:
Is this because customers that are created automatically always start with an incomplete order?

Reply:
Customer records are created with (cart) orders. So if there's a customer record without an order, it must have been created manually.